packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -27,11 +32,46 @@ type CustomLDUser = {
   tenant: string;
 };
 
+type TFetchedGraphQLAdapterFlag = {
+  name: string;
+  value: boolean;
+  reason?: string;
+};
+type TGraphQLAdapterRequestHeaders = {
+  'x-project-key': string;
+  'x-graphql-target': string;
+};
+
 // This value is hard-coded here because we want to make sure that the
 // app uses our account of LD. The value is meant to be public, so there
 // is no need to be concerned about security.
 const ldClientSideIdProduction = '5979d95f6040390cd07b5e01';
 
+combineAdapters.combine([ldAdapter, graphqlAdapter]);
+
+const getRequestHeaders = (props: Props) =>
```

Comment:
apologies if I confuse things.

- the `getRequestHeaders` is supposedly called with [`adapterArgs`](https://github.com/tdeekens/flopflip/blob/8fd148b724394887e99ed9136bb309776a2fc375/packages/types/src/index.ts#L69-L71). 
- Here, we accept `props`.. Where did props come from?

Reply:
Thanks. That's a good catch!

Reply:
ping me when you resolve it ;)

Reply:
Sure. I am doing this on the side. So might take some time :)